Backend/Remora.Discord.API/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -1229,6 +1229,26 @@ private static JsonSerializerOptions AddInteractionObjectConverters(this JsonSer
         options.AddDataObjectConverter<IPartialSelectOption, PartialSelectOption>()
             .WithPropertyName(o => o.IsDefault, "default");
 
+        options.AddDataObjectConverter<IPartialContainerComponent, PartialContainerComponent>()
+            .WithPropertyName(o => o.IsSpoiler, "spoiler");
+
+        options.AddDataObjectConverter<IPartialFileComponent, FileComponent>()
+            .WithPropertyName(o => o.IsSpoiler, "spoiler");
+
+        options.AddDataObjectConverter<IPartialMediaGalleryComponent, PartialMediaGalleryComponent>();
```

Comment:
Should this (and the ones above) have `IncludeWhenSerializing(o => o.Type);` as well?

Reply:
Yes, and these have been rectified.